### PR TITLE
Migration: changes

### PR DESCRIPTION
**Pull request by Paul-tes**
- Created  a new  rails app with the name of **Blog App**
- Added  Postgres database set up.
- Built  project schema.
- Create and run the necessary migration files based ont the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- Added user migration
- Added post migration
- Added comment migration
- Added like migration.

### CODE REVIEW

File: db/migrate/20230816201738_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :post_counter
+      # t.datetime :created_at
+      # t.datetime :updated_at
```
Comment (Ismailco) [inline db/migrate/20230816201738_create_users.rb]: - [x] Kindly make sure to remove all commented code from your code base, it makes your code messy and hard to read

File: README.md [hunk 2]
```diff
@@ -100,81 +64,64 @@ For more information on the importance of a professional README for your reposit
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- Comming soon
```
Comment (Ismailco) [inline README.md:74]: - [x] Kindly consider removing the `Live Demo` section altogether from your README file until you're planning on deploying the project

File: README.md [hunk 3]
```diff
@@ -248,44 +187,23 @@ Feel free to check the [issues page](../../issues/).
...
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
+This project is [MIT](./MIT) licensed.
```
Comment (Ismailco) [inline README.md]: - [x] The License is missing on your project kindly make sure to add a license and fix this link so it's not broken

File: db/migrate/20230816202514_create_posts.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+      # t.datetime :created_at
+      # t.datetime :updated_at
+      t.integer :comments_counter
```
Comment (Ismailco) [inline db/migrate/20230816202514_create_posts.rb]: - [x] Same here kindly make sure to remove commented code

File: db/migrate/20230816202753_create_comments.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: true
+      t.text :text
+      # t.datetime :created_at
+      # t.datetime :updated_at
```
Comment (Ismailco) [inline db/migrate/20230816202753_create_comments.rb]: - [x]  Same here kindly make sure to remove commented code

File: db/migrate/20230816203245_create_likes.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: true
+      # t.datetime :created_at
+      # t.datetime :updated_at
```
Comment (Ismailco) [inline db/migrate/20230816203245_create_likes.rb]: - [x]  Same here kindly make sure to remove commented code

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Paul-tes, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -248,44 +184,23 @@ Feel free to check the [issues page](../../issues/).
...
```
Comment (brainconnect93) [inline README.md]: - [x] As suggested by the previous reviewer, kindly work on the MIT license link as the link is still not working. I realized you have target the file name wrongly, it should be `./MIT.md`

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Paul-tes, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Created migration file for `users`, `posts`, `comments` & `likes` ✔️
- Migrate all file to the schema ✔️ 
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Paul-tes , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**